### PR TITLE
Rework size scaling for hologram scribing

### PR DESCRIPTION
**Pull request by shartse**
The main things this PR addresses are:
1. Set the `width` and `height` of the containing SVG document based on the canvas size
2. Create an inner `viewbox` for compute the `extent` for it based on the bounds of the point set plus some margin
3. Change the margin to be proportional rather than absolute 

With all of these changes, we can use the x and y point locations directly, and trust the outer SVG to scale the image.

For example, with a `canvas_size=10x10cm` and a margin of `0.25` (or 25%), we get SVGs that look like this:
```
<svg height="100" width="100" xmlns="http://www.w3.org/2000/svg">
<svg viewBox="-1.850651 -1.850651 2.701302 2.701302" xmlns="http://www.w3.org/2000/svg">
<circle cx="-0.525731" cy="0.850651" fill="none" r="0" stroke="black" stroke-width="0.005"/>
<circle cx="-0.42058483" cy="0.850651" fill="none" r="0" stroke="black" stroke-width="0.005"/>
<circle cx="-0.31543863" cy="0.850651" fill="none" r="0" stroke="black" stroke-width="0.005"/>
...
</svg>
</svg>
```
test4

Because the extent is computed based on the point set, the object is centered. With a bit of a margin, all the circles fit in the image.

### CODE REVIEW

**Review by samuelselleck — APPROVED** [review]
Looks good! :D

I still believe in the idea of letting some of these things be performed as its own "transformation" step (the thing we talked about with having the model centered + rotated + scaled to fit the canvas before being converted to points), I'm not exactly sure how this fits into that/what parts here should be moved in that case, but we'll take that when we get there!

**Comment by blairfrandeen** [comment]
Sara, as I mentioned yesterday, this change currently breaks the visualizer. I agree that this change is necessary in the long run and I'm okay to merge it and have the visualizer be broken.

In short, you've done with your SVGs what I recommended, which is create a viewbox inside the main `svg` tag. Your output file looks like this:
```xml
<svg height="500" width="500" xmlns="http://www.w3.org/2000/svg">
<svg viewBox="-1.2759765 -1.2759765 2.551953 2.551953" xmlns="http://www.w3.org/2000/svg">
<circle cx="0.850651" cy="0" fill="none" r="-0.13143276" stroke="black" stroke-width="0.005"/>
[...]
```

The current visualizer looks at the first and only the first `svg` tag and sees that it has a height & width of 500. It then picks out all of the circles, which in your example have coordinates and sizes between 0 and 1. The output of the visualizer is then:
```xml
<svg height="500" width="500" xmlns="http://www.w3.org/2000/svg">
<style>
[...]
</style>
<svg viewBox="0 0 500 500" xmlns="http://www.w3.org/2000/svg">
<circle class="inputCircle" cx="0.850651" cy="0" r="-0.13143276" stroke-width="0.049999997"/>
[...]
```
In short, the visualizer doesn't pick up on the fact that all the input circles are in a much smaller viewbox. I will work this morning on figuring out how to deal with that. I've opened Issue #13 to track this.
